### PR TITLE
chapter_4

### PR DESCRIPTION
**Pull request by huyhuynh0296**

### CODE REVIEW

**Comment by huyhuynh0296** [comment]
rubocop_chapter_4

**Comment by huyhuynh0296** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title
+    base_title = "Ruby on Rails Tutorial Sample App"
```
Comment (nvtanh) [inline app/helpers/application_helper.rb]: I18n

File: config/routes.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
-  get 'static_pages/home'
-
-  get 'static_pages/help'
-
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  #root 'static_pages#home'
+  get  'static_pages/home'
+  get  'static_pages/help'
+  get  'static_pages/about'
+  root 'application#hello'
```
Comment (nvtanh) [inline config/routes.rb]: nháy "

**Comment by nvtanh** [comment]
commented

**Comment by huyhuynh0296** [comment]
chapter_4

**Comment by huyhuynh0296** [comment]
chapter_4 finish

**Comment by huyhuynh0296** [comment]
ready

**Pull request merged**